### PR TITLE
AzureAD Updated Detections

### PR DESCRIPTION
**Pull request by mvelazc0**
### Details

This PR updates 25 Azure AD detections to use the latest schema provided by the Splunk Add-on for Microsoft Cloud Services in its latest versions.

### Updated Detections

- Azure AD Global Administrator Role Assigned
- Azure AD Multiple Users Failing To Authenticate From Ip
- Azure AD Service Principal Owner Added
- Azure AD Unusual Number of Failed Authentications From Ip
- Azure AD Service Principal Created
- Azure AD Privileged Role Assigned
- Azure AD Privileged Authentication Administrator Role Assigned
- Azure AD Application Administrator Role Assigned
- Azure AD Multi-Factor Authentication Disabled
- Azure AD External Guest User Invited
- Azure AD User Enabled And Password Reset
- Azure AD Service Principal New Client Credentials
- Azure AD New Federated Domain Added
- Azure AD New Custom Domain Added
- Azure AD Successful Single-Factor Authentication
- Azure AD Authentication Failed During MFA Challenge
- Azure AD Successful PowerShell Authentication
- Azure AD Multiple Failed MFA Requests For User
- Azure AD User ImmutableId Attribute Updated
- Azure Active Directory High Risk Sign-in

### Checklist

- [ ] Validate name matches `<platform>_<mitre att&ck technique>_<short description>` nomenclature
- [ ] [CI/CD](https://github.com/splunk/security_content/actions) jobs passed ✔️ 
- [ ] Validated SPL logic.
- [ ] Validated tags, description, and how to implement.
- [ ] Verified references match analytic.

### CODE REVIEW

File: detections/cloud/azure_active_directory_high_risk_sign_in.yml [hunk 1]
```diff
@@ -10,13 +10,12 @@ description: The following analytic triggers on a high risk sign-in against Azur
   sign-in events using heuristics and machine learning to identify potentially malicious
   events and categorizes them in three categories high, medium and low.
 data_source: []
-search: ' `azuread` body.category=UserRiskEvents body.properties.riskLevel=high |
-  rename body.properties.* as * |  stats values(userPrincipalName) by _time, ipAddress,
+search: ' `azuread` category=UserRiskEvents properties.riskLevel=high |
+  rename properties.* as * |  stats values(userPrincipalName) by _time, ipAddress,
```
Comment (patel-bhavin) [inline detections/cloud/azure_active_directory_high_risk_sign_in.yml]: Write the search as `| stats values(userPrincipalName) as userPrincipalName`. This way we have userPrincipalName in the final table that gets created.

File: detections/cloud/azure_active_directory_high_risk_sign_in.yml [hunk 2]
```diff
@@ -31,7 +30,7 @@ tags:
   asset_type: Azure Active Directory
   confidence: 90
   impact: 60
-  message: A high risk event was identified by Identify Protection for user $body.properties.userPrincipalName$
+  message: A high risk event was identified by Identify Protection for user $properties.userPrincipalName$
```
Comment (patel-bhavin) [inline detections/cloud/azure_active_directory_high_risk_sign_in.yml]: remove `properties. `. Should be only `userPrincipalName`. I believe this applies to a bunch of changes in this PR!

**Review by patel-bhavin — CHANGES_REQUESTED** [review]

File: detections/cloud/azure_active_directory_high_risk_sign_in.yml (hunk 2 above)
Comment (mvelazc0) [reply detections/cloud/azure_active_directory_high_risk_sign_in.yml]: solved, thanks!

File: detections/cloud/azure_active_directory_high_risk_sign_in.yml (hunk 1 above)
Comment (mvelazc0) [reply detections/cloud/azure_active_directory_high_risk_sign_in.yml]: solved, thanks

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**